_overviews/contributors/index.md

```diff
@@ -589,78 +530,28 @@ can browse it at [https://scalacenter.github.io/library-example/](https://scalac
 
 ### Continuous Publication
 
-You need to grant the CI job write access to the Git repository hosting the documentation. This can be achieved
-by creating an SSH key that the CI job can use to push the website to GitHub.
-
-Create an SSH key:
-
-~~~ bash
-$ ssh-keygen -t rsa -b 4096 -C "sbt-site@travis" -f ci/travis-key
-~~~
-
-Make sure to **not** define a passphrase (just leave it empty and press enter), and to add the private
-key (the `ci/travis-key` file) to your `.gitignore`:
+You can extend `.github/workflows/release.yml` to automatically publish documentation to GitHub pages.
+To do so, add another job:
 
-~~~
-ci/secring.asc
-ci/travis-key
-~~~
-
-Add the public key, `ci/travis-key.pub`, in the Deploy Keys section of your GitHub project’s settings page:
-
-![](/resources/images/travis-publishing-key.png)
-
-Make sure you “allow write access” by checking the box.
-
-The private key has to be added to the repository, like we did with the PGP private key. Unfortunately, due
-to a limitation of Travis CI, you can not add several encrypted files. The
-[workaround](https://docs.travis-ci.com/user/encrypting-files/#encrypting-multiple-files) consists in
-creating an archive containing all the files to encrypt. In your case, you want to encrypt the PGP
-key and the SSH key into a single `ci/secrets.tar` file:
-
-~~~ bash
-$ tar cvf ci/secrets.tar ci/secring.asc ci/travis-key
-$ travis encrypt-file ci/secrets.tar ci/secrets.tar.enc --add
-~~~
-
-Make sure to add the `ci/secrets.tar` file to your `.gitignore`:
+```yml
+# .github/workflows/release.yml
+name: #...
 
-~~~
-ci/secring.asc
-ci/travis-key
-ci/secrets.tar
-~~~
-
-Finally, update the `.travis.yml` file to unpack the archive and push the documentation website
-on releases:
-
-~~~ yaml
 jobs:
-  include:
-    - stage: test
-      # Run tests for all Scala versions
-      script: sbt +test
-      name: "Tests"
-      # Check that the documentation can be built
-    - script: sbt makeSite
-      name: "Documentation"
-
-    - stage: deploy
-      if: tag =~ ^v
-      script:
-        # decrypt PGP secret key and GitHub SSH key
-        - openssl aes-256-cbc -K $encrypted_602f530300eb_key -iv $encrypted_602f530300eb_iv -in ci/secrets.tar.enc -out ci/secrets.tar -d
-        - tar xvf ci/secrets.tar
-        # load the key in the ssh-agent
-        - chmod 600 ci/travis-key
-        - eval "$(ssh-agent -s)"
-        - ssh-add ci/travis-key
-        # perform deployment
-        - sbt makeSite +publishSigned sonatypeRelease ghpagesPushSite
-~~~
-
-(Replace the `$encrypted_602f530300eb_key` and `$encrypted_602f530300eb_iv` variables with the ones produced by the
-`travis encrypt-file` command)
+  release: #...
+  publishSite:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v10
```

Comment:
I have a general question about Actions: would it be possible to not run the steps checkout and setup-scala only once and reuse the result in several workflows (e.g., release and publishSite)?

Reply:
AFAIK it is not possible because each job could run on a different worker and therefore we need to re-perform the setup